task5/src/main/java/com/example/demo1/WebAppServlet.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo1;
+import java.io.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+
+@WebServlet(value = "/webapp-servlet")
+public class WebAppServlet extends HttpServlet {
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
```

Comment:
Please consider using constants for static text blocks (this file and all the rest)